geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -130,11 +131,15 @@ public ExecutionHandlerContext(Channel channel,
   }
 
   public ChannelFuture writeToChannel(RedisResponse response) {
-    return channel.writeAndFlush(response.encode(byteBufAllocator), channel.newPromise())
-        .addListener((ChannelFutureListener) f -> {
-          response.afterWrite();
-          logResponse(response, channel.remoteAddress().toString(), f.cause());
-        });
+    return channel
```

Comment:
I don't see any code changes in this block. please revert it

Reply:
done